Apps/Sandcastle/gallery/Cesium OSM Buildings Styling.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Worldwide 3D buildings with rich metadata from OpenStreetMap."
+    />
+    <meta
+      name="cesium-sandcastle-labels"
+      content="Beginner, Showcases, ion Assets"
```

Comment:
Remove the "Showcases" and "ion Assets" tags. Add "3D Tiles".

Reply:
This is what makes it show up when you click on these individual labels in Sandcastle.